src/opcode.rs

```diff
@@ -1375,6 +1375,7 @@ opcode!(
 // === 5.11 ===
 
 opcode!(
+    /// Shut down all or part of a full duplex connection on a socket, equivalent to `shutdown(2)`.
```

Comment:
It would be good to add available kernel version in document.

Reply:
Added, is it worth doing this everywhere? Most docs entries don't have the kernel version explicitly mentioned

Reply:
Yes, all API should have version docs, just like manpage.